lib/trino-hive-formats/pom.xml

```diff
@@ -27,6 +27,12 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
It looks like this dependency is only used inside of `Match#toJson`. If we have sufficient test coverage, it's probably worthwhile to swap that implementation over to jackson to avoid adding another JSON dependency for such a localized use case.